example/spec/backupRestore/BackupRestoreSpec.js

```diff
@@ -1,84 +0,0 @@
-'use strict';
```

Comment:
So did we delete this whole file because it tests the same thing as the api unit test?

Reply:
Yeah I did. Both the unit and integration tests were broken, since the thing didn't work and I didn't see us testing anything differently in integration vs. unit tests.